File: src/main/java/com/graqr/threshr/model/redsky/store/nearby/NearbyStoreRoot.java
package com.graqr.threshr.model.redsky.store.nearby;

import io.micronaut.core.annotation.Introspected;
import io.micronaut.serde.annotation.Serdeable;

@Serdeable
@Introspected
public record NearbyStoreRoot(Data data) {
}

File: src/main/kotlin/com/graqr/threshr/circleStore/model/store/nearby/NearbyStoreRoot.kt
package com.graqr.threshr.circleStore.model.store.nearby

import io.micronaut.core.annotation.Introspected
import io.micronaut.serde.annotation.Serdeable

@Serdeable
@Introspected
data class NearbyStoreRoot(val data: Data) {
}

